middlewares/auth.js

```diff
@@ -0,0 +1,49 @@
+const { checkToken } = require('../helpers/jwt')
+const { user, task } = require('../models')
+
+class Auth {
+    static authentication (req, res, next) {
+        try {
+            let decoded = checkToken(req.headers.access_token)
+            // console.log( decoded);
+            user.findOne({where: {
+                email: decoded.email
+            }})
+            .then( data => {
+                if(!data) {
+                    res.status(401).json({message: 'Please Login First'})
+                } else {
+                    req.user = data
+                    next()
+                }
+            })
+            .catch( err => {
+                res.status(500).json({message: err.message})
+            })
+        } 
+        catch (err) { 
+            res.status(500).json({message: err.message})
+        }
+    }
+
+    static authorization (req, res, next) {
+        const id = +req.params.id
+        const userData = +req.user.id
```

Comment:
```suggestion
        const userId = +req.user.id
```
kan cuman assign id aja 😄